efficientdet/hparams_config.py

```diff
@@ -281,6 +283,13 @@ def default_detection_configs():
   h.dataset_type = None
   h.positives_momentum = None
 
+  # Reduces memory during training
+  h.gradient_checkpointing = False
+  h.gradient_checkpointing_list = ["Add"]
```

Comment:
Could you add a comment to explain what values can be used other than "Add"?

Reply:
Thanks for adding more details.  Could you explain a little bit more:  what's the impact of this list?

If I use ["Add"], does it mean it would automatically checkpoint all "Add" operation?
If I use ['Add', 'Sigmoid'], does it mean it would automatically checkpoint all 'Add' and 'Sigmoid" ops?

If so, what's the pros and cons for adding more ops, and why the default is 'Add'?

Sorry if these questions annoy you, but I am hoping to make it clear as this is a greatly useful feature.  Thanks!